### ISSUE TITLE
How do you count Microsoft Edge browsers for iOS and Android

### ISSUE THREAD

**Issue by rj-david** [Fyrd/caniuse#5473]
Both Microsoft Edge browsers for iOS and Android are version 45.X which are not in any table.

Where do you consider these browsers?

**Comment by atjn** [comment]
Edge on IOS is based on Safari, so you would have to follow the table for `iOS Safari`.
Edge on Android will 99% of the time have support identical to `Chrome for Android`.

Hope this helps!

**Comment by rj-david** [comment]
Thanks, @atjn 

In terms of tools (i.e. those using can-i-use-lite db), this is currently a challenge. Previously all our tools from development, to bundling, to displaying notice of outdated browsers use the same harmonized db. This ms edge cases in ios and android break this and there are now more than 10m downloads of edge at least in android.

Anybody knows where these details mist be included so that it will cascade to the tools?

**Comment by atjn** [comment]
The solution to your problem probably depends on which tool you're using. There are already existing issues, like that Chrome for IOS also needs to be based on `iOS Safari`, so your tool must already have a way of correlating real browser versions with "Caniuse browsers", and that's probably where you need to look to fix this.

When it comes to Edge for Android, you could open an issue and suggest that `Edge for Android` is added to the browser list. Once again, it would also be a pretty safe bet to just base it on `Chrome for Android`.

**Comment by PellegattaSimone** [comment]
`Edge for Android` is currently based on `Chromium 77.0`, while `Chrome for Android` is based on `Chromium 89.0` (the latest version).
Furthermore, it seems that Edge is kind of stuck on that version, since it hasn't been updated for a while now.
This leads to several issues like the lack of many CSS3 functions.

I think that it might be clearer if [caniuse.com](https://caniuse.com/) listed `Edge for Android` as a different browser, or at least if it showed that it is far behind his Google counterpart.
Just to make it clear how much difference there is between the two versions, `Edge for Android` doesn't even implement `edge:\\flags`.

If you have any further information about whether `Edge for Android` will eventually update to `Chromium 89.0` anytime soon, feel free to tell below. I couldn't find anything.

**Comment by GaryHyland** [comment]
Same deal for me, it difficult with my apps browserslist file to indicate Edge for Android (current version 45).

**Comment by PellegattaSimone** [comment]
> Edge on Android will 99% of the time have support identical to `Chrome for Android`.

Not quite... _Edge for Android Stable_ is still based on **Chromium 77.0** (and has been for ages), while _Chrome for Android Stable_ is already at **91.0**.

However, _Edge for Android Dev_ and _Canary_ have already reached version **92.0**, so we can consider these two channels aligned with Google's browser!

**Comment by PellegattaSimone** [comment]
You can check the Chromium version by typing _about:version_ inside the search bar.

**Comment by atjn** [comment]
@PellegattaSimone @GaryHyland  I addressed this in https://github.com/Fyrd/caniuse/issues/5492#issuecomment-799458414
